src/promptflow/tests/sdk_cli_test/recording_utilities/record_storage.py

```diff
@@ -219,6 +226,18 @@ def generator():
 
             output_generator = generator()
             output_type = "generator"
+        elif isinstance(output, Exception):
+            output_value = {
```

Comment:
output_value = output, is this work?

Reply:
it will not, thing might failed when object get unpicklered : 
self = <shelve.DbfilenameShelf object at 0x0000020A326E4290>
key = 'd5f6b487eb8a47e0cf56d9005db0791b9b8cf85b'

    def __getitem__(self, key):
        try:
            value = self.cache[key]
        except KeyError:
            f = BytesIO(self.dict[key.encode(self.keyencoding)])
>           value = Unpickler(f).load()
E           TypeError: APIStatusError.__init__() missing 2 required keyword-only arguments: 'response' and 'body'

C:\Users\chw\anaconda3\envs\promptflow_3.11\Lib\shelve.py:114: TypeError